package.json

```diff
@@ -22,7 +22,7 @@
   },
   "devDependencies": {
     "@superfaceai/ast": "^1.0.0",
-    "@superfaceai/cli": "^1.0.1",
+    "@superfaceai/cli": "1.1.0-beta.0",
```

Comment:
Is this change necessary for this PR, can't it be done in a separate PR?

Reply:
It's mainly to unblock lint in CI 🤷🏻‍♂️